### DISCUSSION TITLE
skynet 的 mongo 实现 (c 部分) 有点老了

### DISCUSSION THREAD

**Discussion by aceyin** [cloudwu/skynet#1877] [General]
最近尝试在项目里面用 mongo  

翻 `skynet` 的源码及 `mongo` [网络协议的官方文档的时候](https://www.mongodb.com/docs/manual/reference/mongodb-wire-protocol/)，发现有很多 OP_CODE 在 mongodb 5.x 的版本中将会被移除，如: 

>  OP_UPDATE, OP_INSERT, OP_QUERY, OP_GET_MORE, OP_DELETE, OP_KILL_CURSORS 等

而 skynet/lua-mongo.c 里面使用的还是这些即将被移除的 OPCODE。

不知道大佬们有计划更新一下吗？ 

（主要是我不懂C，不然可以自己修了，提一个 merge-request）

**Comment by cloudwu** [comment]
你看的哪里的源码？

See https://github.com/cloudwu/skynet/pull/1649

**Reply by aceyin** [reply]
👍

我们用的是 1.5 的版本
看来后面可以考虑升到最新版
